### DISCUSSION TITLE
2014 Bracket

### DISCUSSION THREAD

**Discussion by Sharifk8** [TedJuch/TedTournament#18] [General]
Hey Ted, this isn't directly related to your GitHub, but I came across your 2014 March Madness tournament while looking for ways to automate a scoring system for all the particpants using a Master Spreadsheet Key. Your video was really helpful and I wanted to use your templates linked below the videos but it requires access. I requested access and would really love if you were able to grant it to me. I've been running fun tournaments with my friends the past year and I think your templates could really elevate the experience! Please let me know what you think. Thank you

**Comment by TedJuch** [comment]
Hi @Sharifk8 I've updated the YouTube video description to include the most up to date template. You can find them here:

When you click on the links below Sheets will automatically make your own private copy of the template. No need to request access.

Individual Bracket Template: https://docs.google.com/spreadsheets/d/1izjBEQ_FIU0dJ2Z1exWMY2FwpmDP6AqHYxlldD6xhO4/copy

Group Template: https://docs.google.com/spreadsheets/d/1UBEQnmpWKKHPXu4Y3xmUAlxWR4Oo9jPAXCfL_e-gMT8/copy

**Reply by Sharifk8** [reply]
Thank you so much! I really appreciate you helping me out. The brackets/tournaments you make are really impressive and useful.

**Reply by Sharifk8** [reply]
Hi @TedJuch , I'm having trouble creating a custom bracket. I have been able to enter all the custom teams and get the Custom Key, but I'm having trouble with number 7-9 on the instructions: 
"Open the hidden tab called Custom Key Authorization
Hover over A1 and click on Allow Access
Your custom data source should now be synced to your bracket group template."
I have been using the 2023 bracket group template and trying to add the custom key to it, but it's not working. I was wondering if it is because I need the 2020 bracket group template in order for it to work? Please let me know what you think. Thanks!

**Comment by TedJuch** [comment]
Let me take a look at the templates and get back to you. Thanks!

**Comment by TedJuch** [comment]
@Sharifk8 Can you share the files with me at ted.juch@gmail.com? I'll need edit access on both to see what the issue is.

**Reply by Sharifk8** [reply]
@TedJuch I just shared both of them with you. I was able to successfully add the custom key to the group bracket. I also shared an individual bracket which I added the custom key to and was able to authorize but I don't think the brackets are synced up. Please let me know what you think. Thank you!

**Reply by TedJuch** [reply]
@Sharifk8 I think I've fixed everything. The individual and group brackets were looking for a 2023 Custom Bracket tab on the Custom Bracket Spreadsheet but they were named as 2020 Custom Bracket. This was initially developed during the pandemic when there wasn't an NCAA tournament so there are a few things that were hard-coded for 2020. There was also a formula bug with determining winners before the games were complete which I fixed. Take a look and let me know if you still see issues.

**Reply by Sharifk8** [reply]
@TedJuch the brackets are synced, and the custom key is working! I no longer see any issues, thank you for fixing that, I really appreciate it

**Comment by TedJuch** [comment]
@Sharifk8 glad to hear it! I'm going to close this comment.
